src/pages/ViewScholarship.jsx

```diff
@@ -83,6 +84,7 @@ export default function ViewScholarship() {
       />
 
       <ScholarshipCard scholarship={scholarship} style="detail" />
+      <ShowMoreScholarships />
```

Comment:
Maybe something like this to hide the current scholarship from the results.
```suggestion
      <ShowMoreScholarships currentId={scholarship.id} />
```